index.html

```diff
@@ -1,3 +1,66 @@
-﻿
+<!doctype html>
+<style>
+    body{
+      font-family: Arial, sans-serif;
+      background-image: url("./img/bg-img.jpg");
+      background-repeat: repeat;
+      padding: 10px;
+    }
+</style>
 
+<html lang="en">
+  <head>
+    <meta name="description" content="Demo project using HTML, CSS, & JavaScript">
+    <title>Grandma's Cookies, yo</title>
+  </head>
 
+  <body>
+    <img src="./img/cookies.jpg" alt="cookies" height="283" width="566">
+
+    <header>
+      <h1>The Best Chocolate Chip Cookies</h1>
+      <p>Recipe by: Grandma <br> Prep Time: 45 Min</p>
```

Comment:
Why a `<br>` instead of 2 paragraphs?

Reply:
that's probably better practice but I think I was just lazy cause a br tag is less to type